src/components/Posts/ComposePost.tsx

```diff
@@ -77,17 +88,17 @@ const ComposePost = ({ placeholder }: ComposePostProps) => {
           />
           <Box sx={styles.postActions}>
             <Stack direction="row">
-              <IconButton size="small">
+              <IconButton size="small" color="primary">
```

Comment:
What was the rationale for making the action buttons in these compose post/reply components always green? Should we have all of our buttons look and act the same instead?

Reply:
I was basically cloning twitter because they don't have it gray at any point

Reply:
We ended up making all of our icon buttons follow the same pattern